packages/@sanity/default-layout/src/navbar/presence/PresenceMenu.tsx

```diff
@@ -0,0 +1,114 @@
+// @todo: remove the following line when part imports has been removed from this file
+///<reference types="@sanity/types/parts" />
```

Comment:
This file no longer contains parts, so this TS reference + comment can be removed.